runtime/src/iree/vm/native_module_benchmark.cc

```diff
@@ -4,8 +4,10 @@
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-#include "benchmark/benchmark.h"
+#if !IREE_HAS_NOP_BENCHMARK_LIB
```

Comment:
I'm not so sure about `#if` guarding every `_benchmark.cc` file like this. Can filtering be done in the CMake `iree_cc_binary_benchmark` function instead? Only a few files are affected, but it's still weird boilerplate...

If keeping this, how about inverting the define/check to remove the double negative? That way makes it more obvious to me what each source file should be doing:

```c++
#if IREE_HAS_BENCHMARK_IMPL

// ... file contents ...

#endif
```

Reply:
The double negative is just preference: it defaults to enabled. If the define ever changes or the machinery messes up, it will attempt to compile and fail (versus being silently excluded forever).